ecosystem.config.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+  /**
+   * Application configuration section
+   * http://pm2.keymetrics.io/docs/usage/application-declaration/
+   */
+  apps : [
+
+    // First instance of microservice
+    {
+      name      : "data-transfer",
+      script    : "bin/data-transfer",
+      interpreter: "python",
+      env: {
```

Comment:
These settings fail when run 'out of the box'. I suggest changing the values to match the defaults / Docker dependencies.